integrationTests/package.json

```diff
@@ -8,9 +8,10 @@
     "depcheck": "depcheck",
     "lint": "eslint . --ext .ts,.js --quiet --fix",
     "pretest": "npm run lint && npm run depcheck",
-    "test:ts": "nyc mocha \"tests/*.test.ts\"",
-    "test:js": "nyc mocha \"tests/*.test.js\"",
+    "test:ts": "nyc mocha \"tests/basic/*.test.ts\"",
+    "test:js": "nyc mocha \"tests/basic/*.test.js\"",
     "test": "npm run test:ts && npm run test:js",
+    "test:redis": "nyc mocha \"tests/redis/*.test.js\"",
```

Comment:
Is there a different name that could be used here instead of test:redis? It implies redis is being started, see my earlier comment for an exampe of the confusion.

Also, if the tests are agnostic of the caching tech then the name shouldn't be included as a part of the test.

Reply:
unfortunately tests are not agnostic to the caching tech, redis needs redis docker dependency

Reply:
Our integration tests fail if we use a different caching tool?

Reply:
Any new tool needs changes to tests